tests/test_tags.py

```diff
@@ -146,6 +146,6 @@ def test_metablock_with_modifications(render_template):
 
         <a href="">supercomputers</a> , period <-> Jane & John Doe.
     {% endmetablock %}"""
-    expected_output = 'The world&#39;s &quot;fastest&quot; supercomputers, period &lt;-&gt; Jane &amp; John Doe.'
+    expected_output = 'The world&#x27;s &quot;fastest&quot; supercomputers, period &lt;-&gt; Jane &amp; John Doe.'
```

Comment:
This appears to be a change in Wagtail's rich text processor. The previous output was actually wrong, `&#39;` is not an apostrophe.